scripts/linters/pylint_extensions_test.py

```diff
@@ -2844,3 +2844,98 @@ class ClassName(dummy_class):
 
         with self.checker_test_object.assertAddsMessages(message):
             temp_file.close()
+
+
+class SingleLinePragmaCheckerTests(unittest.TestCase):
+
+    def setUp(self):
+        super(SingleLinePragmaCheckerTests, self).setUp()
+        self.checker_test_object = testutils.CheckerTestCase()
+        self.checker_test_object.CHECKER_CLASS = (
+            pylint_extensions.SingleLinePragmaChecker)
+        self.checker_test_object.setup_method()
+
+    def test_pragma_for_muliline(self):
+        node_pragma_for_multiline = astroid.scoped_nodes.Module(
+            name='test',
+            doc='Custom test')
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                    # pylint: disable=invalid-name
+                    def funcName():
+                        \"\"\" # pylint: disable=test-purpose\"\"\"
+                        pass
+                    # pylint: enable=invalid-name
+                """)
+        node_pragma_for_multiline.file = filename
+        node_pragma_for_multiline.path = filename
+
+        self.checker_test_object.checker.process_tokens(
+            utils.tokenize_module(node_pragma_for_multiline))
+
+        message1 = testutils.Message(
+            msg_id='single-line-pragma',
+            line=2)
+
+        message2 = testutils.Message(
+            msg_id='single-line-pragma',
+            line=6)
+
+        with self.checker_test_object.assertAddsMessages(
+            message1, message2):
+            temp_file.close()
+
+    def test_enable_single_line_pragma_for_muliline(self):
+        node_enable_single_line_pragma_for_muliline = (
+            astroid.scoped_nodes.Module(name='test', doc='Custom test'))
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                    # pylint: disable=single-line-pragma
```

Comment:
Add a test for this case:

`# pylint: disable=invalid-name, single-line-pragma`

Reply:
Done.